st2common/st2common/validators/api/action.py

```diff
@@ -58,18 +60,27 @@ def _is_valid_pack(pack):
 
 
 def _validate_parameters(action_params=None, runner_params=None):
-    for param, action_param_meta in six.iteritems(action_params):
+    for action_param, action_param_meta in six.iteritems(action_params):
+        # Check if overridden runner parameters are permitted.
+        if action_param in runner_params:
+            for action_param_attr, value in six.iteritems(action_param_meta):
```

Comment:
It looks like similar code is duplicated in two functions :)

It would be great if you could refactor it in a separate function (e.g. `is_parameter_overridable` or similar). In addition to that, it would make the code a bit easier to read / follow.